rust/log_viewer/test.json

```diff
@@ -0,0 +1,20 @@
+{"pid": 23, "time": "2023-07-10T11:11:11", "level":"error", "recv_pid":"this is a message", "message":"this is a log message to be viewed"}
```

Comment:
this was an accident and needs to be deleted

Reply:
or turned into a unittest :)